calypso/mod.go

```diff
@@ -0,0 +1,34 @@
+package calypso
+
+import (
+	"go.dedis.ch/dela/crypto"
+	"go.dedis.ch/dela/ledger/arc"
+	"go.dedis.ch/kyber/v3"
+)
+
+// Calypso defines the primitives to run a Calypso app. It is mainly a wrapper
+// arround DKG that provides a storage and authorization layer.
+type Calypso interface {
```

Comment:
I guess you can call that `PrivateStorage` as Calypso is one possible implementation of a private storage.